frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -379,3 +379,49 @@ export const fetchMineDocuments = (mineGuid) => (dispatch) => {
       dispatch(hideLoading());
     });
 };
+
+// MineVerifcationStatus
+export const fetchMineVerifiedStatuses = (user_id) => (dispatch) => {
+  const params = user_id ? { user_id } : null;
```

Comment:
it looks like you're doing the null check inside `API.MINE_VERIFIED_STATUSES` as well, 
```
export const MINE_VERIFIED_STATUSES = (params) =>
  params ? `/mines/verified-status?${queryString.stringify(params)}` : "/mines/verified-status";
```

is `user_id` the only expected param? If so, this could be simplified and user_id could be passed directly in without the above const.

```
export const MINE_VERIFIED_STATUSES = (userID) =>
  userID ? `/mines/verified-status?user_id=${userID}` : "/mines/verified-status";
```